load-testing/audit-service/sendLoginMessages.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#
+# Populates the SQS queue with couple of 100 messages for audit service load testing
+#
+
+export AWS_DEFAULT_REGION=us-east-1
+
+awsURL='https://sqs.us-east-1.amazonaws.com/707767160287'
+sqsURL='qaplanetv2--qa-niaid--audit-sqs'
```

Comment:
like @themarcelor mentioned, an environment variable for the SQS URL could be useful for the future. maybe for the AWS URL too? unless we always run this in the same QA account

Reply:
I have parametrized the URL in the latest commit